fusor-ember-cli/server/mocks/deployment-plan.js

```diff
@@ -0,0 +1,526 @@
+                        },
+                        "description": null
+                    }
+                ],
+                "default": null,
+                "description": null,
+                "hidden": null,
+                "label": null,
+                "name": "Controller-1::count",
+                "parameter_type": "number",
+                "value": "1"
+            },
+            {
+                "constraints": [
+                    {
+                        "constraint_type": "range",
+                        "definition": {
+                            "min": "0"
+                        },
+                        "description": null
+                    }
+                ],
+                "default": null,
+                "description": null,
+                "hidden": null,
+                "label": null,
+                "name": "Ceph-Storage-1::count",
+                "parameter_type": "number",
+                "value": "0"
+            },
+            {
+                "constraints": [
+                    {
+                        "constraint_type": "range",
+                        "definition": {
+                            "min": "0"
+                        },
+                        "description": null
+                    }
+                ],
+                "default": null,
+                "description": null,
+                "hidden": null,
+                "label": null,
+                "name": "Compute-1::count",
+                "parameter_type": "number",
+                "value": "1"
+            },
+            {
+                "constraints": [
+                    {
+                        "constraint_type": "range",
+                        "definition": {
+                            "min": "0"
+                        },
+                        "description": null
+                    }
+                ],
+                "default": null,
+                "description": null,
+                "hidden": null,
+                "label": null,
+                "name": "Swift-Storage-1::count",
+                "parameter_type": "number",
+                "value": "0"
+            },
+            {
+                "constraints": [
+                    {
+                        "constraint_type": "range",
+                        "definition": {
+                            "min": "0"
+                        },
+                        "description": null
+                    }
+                ],
+                "default": null,
+                "description": null,
+                "hidden": null,
+                "label": null,
+                "name": "Cinder-Storage-1::count",
+                "parameter_type": "number",
+                "value": "0"
+            },
+        ],
+        "updated_at": null,
+        "uuid": "5393d0c1-001b-4b4b-99ed-dee163a8200f"
+      }
+    });
+  });
+
+  app.use('/fusor/api222/openstack/deployments', deploymentPlanRouter);
```

Comment:
a non-existing path, such as fusor/api222 is what makes the mocks "disabled". The commit that I wrote in the README is what enables the mocks in development mode using ember-cli
https://github.com/fusor/fusor/commit/367fbb466c2a7a14f1b12c1f3a4ee9f3f15bb6bc